pkg/api/bzz_test.go

```diff
@@ -1090,3 +1090,25 @@ func TestDirectUploadBzz(t *testing.T) {
 		}),
 	)
 }
+
```

Comment:
I suppose that this test will be made functional. Maybe to add also the quotes escaping test in it?